django/otto/management/commands/delete_unused_libraries.py

```diff
@@ -6,6 +6,7 @@
 from django_extensions.management.utils import signalcommand
 
 from chat.models import Library
+from otto.settings import CHAT_RETENTION_DAYS
```

Comment:
shouldn't this be LIBRARY_RETENTION_DAYS?

But in the delete_old_chats if you could please change it so it uses this constant - thanks!